docs/fastdds/env_vars/env_vars.rst

```diff
@@ -2,3 +2,26 @@
 
 Environment variables
 =====================
+
```

Comment:
May I be so bold as to ask for some snippets (windows and linux) to see then in action?

Reply:
Are we assuming a bash console on windows? If we are, both examples would be the same, (except the file name conventions). I gave it a try, you tel me if you like it.